docs/recipes/WritingTests.md

```diff
@@ -465,8 +465,103 @@ it('passes dispatch and getState', () => {
 
 In some cases, you will need to modify the `create` function to use different mock implementations of `getState` and `next`.
 
+### Testing with Storybook
+
+Integrating [Storybook](https://storybook.js.org/) with your redux-based application is very straightforward. The main requirement is to make sure that your stories are wrapped with `Provider` so that the components have access to the store.
+
+> ##### Note on `connect`ed components
+>
+> If your components are all `connect`-based and not yet utilizing hooks, the follow example is still relevant to you, but you can always pass props into the component instead of relying on setting the `preloadedState` as shown below. There is a complete example of that approach in the [Storybook data documentation](https://www.learnstorybook.com/intro-to-storybook/react/en/data/).
```

Comment:
```suggestion
> If your components are all `connect`-based and not yet utilizing hooks, the following example is still relevant to you, but you can always pass props into the component instead of relying on setting the `preloadedState` as shown below. There is a complete example of that approach in the [Storybook data documentation](https://www.learnstorybook.com/intro-to-storybook/react/en/data/).
```
I'm also not sure if the `and not yet utilizing hooks` portion is necessary here

Reply:
Good catch, and agreed :)